src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,50 @@
-export const Tabs = () => {};
+import React from 'react';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab | undefined) => void,
+};
+
+export const Tabs:React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const handleClick = (target: EventTarget & HTMLAnchorElement) => {
+    const currTabId = target.getAttribute('href')?.slice(1);
+
+    if (currTabId !== selectedTabId) {
+      onTabSelected(tabs.find(el => el.id === currTabId));
```

Comment:
It's better to use a separate function to find the tab and pass its ID to the onTabSelected callback.

Reply:
Moreover you use it twice in this component